### DISCUSSION TITLE
How to find YouTube domains?

### DISCUSSION THREAD

**Discussion by SwissTemp** [Seji64/SniDust#56] [Q&A]
Hi

I‘d like to use SniDust for YouTube adblocking. If run on a not monetized country like Albania, YouTube won‘t show any ads. But how to find out all used YouTube domains?

Regards

**Comment by Seji64** [comment]
I googled a bit and added a list for you. However - i don't think youtube can be tricked so easily. i think this is only possible at browser level and not at dns level. However, I am curious to see what comes out of your attempt :-)

**Reply by SwissTemp** [reply]
Thanks, will try! It works for sure, I'm using that approach using external DNS service (ControlD), where I redirect all YouTube traffic to Albania. They have pre-defined services where you can alter them, and it works perfectly fine. But I'd like to use my own server.
